Homework-Functions/src/sciit/homework/functions/ex3/WhatTimeIsIt.java

```diff
@@ -0,0 +1,36 @@
+package sciit.homework.functions.ex3;
+
+import java.util.Scanner;
+
+public class WhatTimeIsIt {
+    public static void main(String[] args) {
+
+        System.out.println("What time is it?");
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Insert the number that represent the hour");
+        int hour = sc.nextInt();
+
+        Scanner sc1 = new Scanner(System.in);
+        System.out.println("Insert the number that represent the minutes");
+        int minutes = sc.nextInt();
+
+        System.out.println(checkHourMinutesFormat(hour, minutes));
+    }
+
+    public static String checkHourMinutesFormat(int hour, int minutes) {
```

Comment:
Metoda ar fi trebuit sa faca verificarea daca datele introduse sunt valide si sa returneze `boolean`, `true` daca datele sunt valide si `false` daca nu, iar verificarea rezultatului si afisarea mesajului de `Incorrect time` ar fi trebuit facute in metoda `main`.